src/main/java/com/telecom/antiphishing/integration/model/EvaluateUriRequest.java

```diff
@@ -0,0 +1,29 @@
+package com.telecom.antiphishing.integration.model;
+
+import com.telecom.antiphishing.integration.enums.ThreatType;
+import lombok.Data;
+
+import java.io.Serializable;
+import java.util.List;
+
+/***
+ *
+ * More details can be found here:
+ * https://cloud.google.com/web-risk/docs/reference/rest/v1eap1/TopLevel/evaluateUri#request-body
+ *
+ * @author - Kiryl Karpuk
+ */
+@Data
+public class EvaluateUriRequest implements Serializable {
+
+    private String uri;
```

Comment:
Czy moglibyśmy zarówno tutaj jak i w innych miejscach używać typu `URI` do trzymania `uri`?

Reply:
Tak, masz rację. Poprawi to czytelność i ochroni przed dowolnym stringiem zamiast uri